spec/cli/commands/ip_spec.rb

```diff
@@ -5,62 +5,119 @@
 describe Ronin::CLI::Commands::Ip do
   include_examples "man_page"
 
-  let(:ipcmd) { Ronin::CLI::Commands::Ip.new}
+  subject { Ronin::CLI::Commands::Ip.new }
   let(:localhost) { "127.0.0.1" }
 
-  describe "-r" do
-    it "will reverse an IPv4 address" do
-      ipcmd.options[:reverse] = true
-      expect{ ipcmd.process_value(localhost) }.to output("1.0.0.127.in-addr.arpa\n").to_stdout
+  describe "#process_value" do
+
+    describe "-r" do
+      it "will reverse an IPv4 address" do
+        subject.options[:reverse] = true
+        expect{ subject.process_value(localhost) }.to output("1.0.0.127.in-addr.arpa\n").to_stdout
+      end
     end
-  end
 
-  describe "--hex" do
-    it "will convert an IPv4 address to hexadecimal" do
-      ipcmd.options[:hex] = true
-      expect{ ipcmd.process_value(localhost) }.to output("0x7f000001\n").to_stdout
+    describe "--hex" do
```

Comment:
You're going to hate me for splitting hairs here, but usually it's a good idea to add some "when --foo option is given" text here, so the rspec output reads better.

While technically `context` is just an alias for `describe`, I use `context` for edge-cases within the `describe` (which describes a method or a class/module).

Reply:
Makes total sense.  I'll adjust accordingly.  Thanks!